src/components/settings/LocationSettings.js

```diff
@@ -41,7 +42,8 @@ export const LocationSettings = () => {
   const {
     locationService = systemPermission.status !== Location.PermissionStatus.DENIED,
     alternativePosition,
-    defaultAlternativePosition
+    defaultAlternativePosition,
+    showAlternativeLocationButton = false
```

Comment:
normally we show this section, so the default should be `true`, right? otherwise we would need to add this entry in all instances.

Reply:
That's right. I used it for the test phase and I forgot.